### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by heowc**
Fix typo

### CODE REVIEW

File: core/src/main/java/com/linecorp/armeria/client/Endpoint.java [hunk 1]
```diff
@@ -477,7 +477,7 @@ private Endpoint withIpAddr(String ipAddr, StandardProtocolFamily ipFamily) {
     }
 
     /**
-     * Returns a new host endpoint with the {@linkplain InetAddress#getHostAddress()} IP address} of
+     * Returns a new host endpoint with the {@linkplain InetAddress#getHostAddress()} IP address of
```
Comment (ikhoon) [inline core/src/main/java/com/linecorp/armeria/client/Endpoint.java:480]:
```suggestion
     * Returns a new host endpoint with the {@linkplain InetAddress#getHostAddress() IP address} of
```

File: core/src/main/java/com/linecorp/armeria/client/redirect/UnexpectedProtocolRedirectException.java [hunk 2]
```diff
@@ -27,7 +27,7 @@
 import com.linecorp.armeria.common.annotation.UnstableApi;
 
 /**
- * An exception indicating that the {@linkplain URI#getScheme()} session protocol} of the redirection URI
+ * An exception indicating that the {@linkplain URI#getScheme()} session protocol of the redirection URI
```
Comment (ikhoon) [inline core/src/main/java/com/linecorp/armeria/client/redirect/UnexpectedProtocolRedirectException.java:30]:
```suggestion
 * An exception indicating that the {@linkplain URI#getScheme() session protocol} of the redirection URI
```

File: core/src/main/java/com/linecorp/armeria/common/logging/BuiltInProperty.java [hunk 3]
```diff
@@ -323,7 +323,7 @@
     }),
 
     /**
-     * {@code "tls.proto"} - the current {@linkplain SSLSession#getProtocol()} TLS protocol}.
+     * {@code "tls.proto"} - the current {@linkplain SSLSession#getProtocol()} TLS protocol.
```
Comment (ikhoon) [inline core/src/main/java/com/linecorp/armeria/common/logging/BuiltInProperty.java:326]:
```suggestion
     * {@code "tls.proto"} - the current {@linkplain SSLSession#getProtocol() TLS protocol}.
```

**Review by ikhoon — APPROVED** [review]
Thanks!

**Review by minwoox — APPROVED** [review]
Thanks. 😄

**Review by trustin — APPROVED** [review]
Thanks, @heowc 🙇

**Pull request merged**